files/conf/hardening.conf

```diff
@@ -18,3 +18,9 @@ net.ipv4.ip_local_reserved_ports=30000-32767
 net.ipv4.conf.all.rp_filter = 1
 net.ipv4.conf.all.arp_ignore = 1
 net.ipv4.conf.all.arp_announce = 2
+
+# These are required for the kubelet '--protect-kernel-defaults' flag
+# See https://github.com/giantswarm/giantswarm/issues/13587
+vm.overcommit_memory=1
+kernel.panic=10
+kernel.panic_on_oops=1
```

Comment:
These are required for the `--protect-kernel-defaults` flag. They settings and consequences are described in [the issue](https://github.com/giantswarm/giantswarm/issues/13587)